prospero-cli/src/main/resources/UsageMessages.properties

```diff
@@ -51,6 +51,10 @@ prospero.channel.remove.usage.header = Unsubscribe the installation from a chann
 prospero.channel.initialize.usage.header = Add a custom channel to be used by the server
 prospero.channel.promote.usage.header = Promote a bundle of artifacts to a custom repository
 
+prospero.clone.usage.header = Manage the installation metadata.
```

Comment:
Maybe something like:
```
prospero.clone.usage.header=Operations to export the installation metadata and use it to recreate installation
prospero.clone.export.usage.header=Export all metadata needed to recreate installation
prospero.clone.import.usage.header=Recreate a server based on exported metadata
```

Actually thinking about it maybe the `import` operation should be called `recreate` or `restore`? WDYT?

Reply:
I prefer `restore` which sounds more like creating a server from an existing metadata.